### ISSUE TITLE
[Quest] Betrayal

### ISSUE THREAD

**Issue by QuinnStoltz** [WoWManiaUK/Redemption#2849]
Doing the quest Betrayal, killing Drakuru drops a skull at his feet but the skull cannot be looted. There are several other skulls left on the altar (I assume these are from other people doing the quest and not getting cleaned up). 

It sounds like this is issue is hit or miss and affects some people but after polling world chat, several other people had the same issue. For some of those people, clearing their client's cache fixed it but I did not try that. I was able to repeat this issue (unfortunately) twice in a row and it kept me from completing the quest.

**Comment by TenmanWoW** [comment]
Quest - 12713 Betrayal
Quest item - 40425  Drakuru's Skull

Quest items provided -

Scepter of Domination 39664 
Ensorcelled Choker 38699 

I ran this 7 times, with different classes on both factions. Under the effect of the ensorcelled choker, a dialogue begins, the disguise fails, combat ensues with Drakuru. After defeating him, the LK appears and kills Drakuru, at which time the skull appears and is completely lootable.  

@QuinnStoltz 

Did you begin the event disguised and kept that disguise until it failed during the dialogue? What class did you attempt it as? I would be interested to know what might be causing your issue, for I am unable to reproduce it.


![wowscrnshot_110918_233625](https://user-images.githubusercontent.com/25806987/48297927-34a1e100-e479-11e8-8df8-45e9608f64ae.jpg)
![wowscrnshot_110918_233614](https://user-images.githubusercontent.com/25806987/48297929-34a1e100-e479-11e8-99c0-d2fa18927d4d.jpg)
![wowscrnshot_110918_233611](https://user-images.githubusercontent.com/25806987/48297930-34a1e100-e479-11e8-9ee0-30e76b3b910c.jpg)

**Comment by QuinnStoltz** [comment]
Hi Tenman,

I was playing on my lvl 75 Worgen Druid at the time. I use the choker and gained the diguise as normal. I replicated the issue once yesterday and twice the day before. This was very recent and definitely an issue.

As per normal, the diguise fails. Then, I used the scepter to control one of the big hunky dudes. This wears off when Drakuru gets to about 10% health and I have to enslave another one to finish him off. At which point, Lich King appears and kills Drakuru. 

During the entire fight, there are many skills laying on the ground that appear with little gear icons BUT I was unable to loot any of these. Once Drakuru is slain by the LK, he drops his own skull. For me, this skull had the same result as clicking on the many other skulls scattered around and after the use-time channel, didn't let me loot the item. 

At this point, I contacted Saintz and he suggested I clear my client cache and try again but I told him it was not worth redoing for no guarantee it would work for me. At this point I am just trying to document the issue so it gets resolved for the future. 

Best,
Mossclaw
